.circleci/config.yml

```diff
@@ -296,8 +299,9 @@ commands:
             --durations=0 \
             --master-host="<<parameters.master-host>>" \
             --master-port="<<parameters.master-port>>" \
+            -o junit_family=xunit1 \
```

Comment:
This is the active ingredient -- the change above is for information and the one below is a matter of shell style.